__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data/index")
+const db = require("../db/connection")
+const request = require("supertest")
+const app = require("../app")
+
+beforeEach(() => seed(data))
+afterAll(() => db.end())
+
+describe("GET /api/categories", () => {
+    it('should return status 200, responds with array of category objects with properties slug and description', () => {
+        return request(app)
+            .get("/api/categories")
+            .expect(200)
+            .then((response) => {
+                const expectedCategories = [
+                    { slug: 'euro game', description: 'Abstact games that involve little luck' },
+                    {
+                      slug: 'social deduction',
+                      description: "Players attempt to uncover each other's hidden role"
+                    },
+                    { slug: 'dexterity', description: 'Games involving physical skill' },
+                    { slug: "children's games", description: 'Games suitable for children' }
+                  ]
+                expect(response.body.categories).toEqual(expectedCategories)
+            })
```

Comment:
Technically this does the job, but you wouldn't want to do this normally. Ideally you want to be iterating over the response and expecting each element of the response array to have a certain structure.